packages/messaging-api-slack/README.md

```diff
@@ -86,6 +86,30 @@ client.postMessage('C8763', 'Hello!');
 client.postMessage('C8763', 'Hello!', { as_user: true });
 ```
 
+`messaging-api-slack` will automatically stringify the `attachments` field for you.
```

Comment:
How about "Or you can attach additional attachments, `messaging-api-slack` will automatically stringify the `attachments` field for you."

Reply:
I'd like
"If you want to send message with `attachments`, `messaging-api-slack` will automatically stringify the `attachments` field for you."
as it's part of the options, not so additional.